.codesandbox/sandbox/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  getSolidDataset,
+  getThing,
+  getStringNoLocale,
+} from "@inrupt/solid-client";
+import { foaf } from "rdf-namespaces";
+
+getSolidDataset("https://vincentt.inrupt.net/profile/card").then(
```

Comment:
Is it okay to use your own profile in this, or would you rather point to a dummy ?

Reply:
My profile is public, so I don't see the harm, but I guess it's better practice to use a dummy. I'll set something up.

Reply:
2be5ab93332682428d1dc7bd6f6bb438ae19d39d